lib/human_readable_id.rb

```diff
@@ -0,0 +1,26 @@
+class HumanReadableId
+  ID_LENGTH = 8
+  MAX_NUMBER = Base32::Crockford.decode('Z' * ID_LENGTH) # Largest 8 digit number
+
+  def self.update_unique_id(klass, primary_key, column)
+    ActiveRecord::Base.transaction(requires_new: true) do
```

Comment:
Maybe add a comment about the ```requires_new```.
My assumption is that when this is ran within the booking request creator it's already within a transaction but we may run this in another context without a transaction and this would ensure there is a transaction?